ush/wave_grid_interp_sbs.sh

```diff
@@ -133,17 +133,18 @@ source "${USHgfs}/preamble.sh"
 
 # 1.b Run interpolation code
 
+  export pgm="${NET,,}_ww3_gint.x"
+  . prep_step
+
   set +x
-  echo "   Run ww3_gint
-  echo "   Executing ${EXECgfs}/ww3_gint
+  echo "   Executing ${pgm}"
   set_trace
 
-  export pgm=ww3_gint;. prep_step
-  ${EXECgfs}/ww3_gint 1> gint.${grdID}.out 2>&1
+  "${EXECgfs}/${pgm}" 1> gint.${grdID}.out 2>&1
   export err=$?;err_chk
 
 # Write interpolation file to main TEMP dir area if not there yet
-  if [ "wht_OK" = 'no' ]
+  if [ "wht_OK" = 'no' ]  # FIXME: This is never going to evaluate to true, wht_OK is a string and needs to be ${wht_OK}
```

Comment:
fascinating b/c i'm pretty sure we copy these in....  we should maybe take a closer look at what's going on here

Reply:
not sure when/where/how this got turned to this.

Reply:
Okay - looking closer, this is "once created, use again" but we do copy the file up on line 128 if it's in the fix file -- which is what i knew we were doing.

Reply:
If there is a suggestion that I should take, please feel free to suggest it, and I can include it in this changeset.

Reply:
Pretty sure this should just be as you said: ${wht_OK}